databricks/koalas/missing/indexes.py

```diff
@@ -81,6 +81,7 @@ class _MissingPandasLikeIndex(object):
     set_value = unsupported_function('set_value')
     slice_indexer = unsupported_function('slice_indexer')
     slice_locs = unsupported_function('slice_locs')
+    sort = unsupported_function('sort')
     sort_values = unsupported_function('sort_values')
```

Comment:
Something went wrong with these lines.

Reply:
oops.. sorry I was in a hurry since the morning meeting. 😅 
going to fix this right away, thanks for the comment!! 👍